tests/test_engine.py

```diff
@@ -181,6 +182,27 @@ def test_env_eval(self):
             if "BZT_ENV_TEST_UNSET" in os.environ:
                 os.environ.pop("BZT_ENV_TEST_UNSET")
 
+    def test_nested_env_eval(self):
+        try:
+            self.obj.config.merge({
+                "settings": {
+                    "env": {"FOO": "${BAR}/aaa/bbb", "FOOBAR": "eee", "BAR": "${BAZ}/ccc", "BAZ": "${FOOBAR}/ddd"}
+                }})
```

Comment:
in this test I tried to add `"ART": "${FOO}===${TAURUS_ARTIFACTS_DIR}"` to the env config and results were
`'${BAR}/aaa/bbb===/home/taras/Projects/taurus/build/test/2020-05-04_10-26-19.584147'` (it's in os.environ, self.obj.confg['settings']['env']['ART'] contains correct value `'eee/ddd/ccc/aaa/bbb===/home/taras/Projects/taurus/build/test/2020-05-04_10-26-19.584147'`)
Looks like substitution of nested BAR doesn't work, does it?

Reply:
Thanks, I forgot to expand OS env vars. It should be fixed now.